test.js

```diff
@@ -109,3 +90,9 @@ it('should error if cannot get repository', function() {
     getPkgRepo({});
   });
 });
+
+it('should parse github enterprise http url', function() {
+  var url = 'http://github.mycompany.dev/user/myRepo';
```

Comment:
I realise other tests only test 
`repo.type` and `repo.browse()`. Feel free to add a few more :)